test/rules/no-magic-numbers/custom/test.ts.lint

```diff
@@ -1,3 +1,4 @@
+parseInt('123', 10);
```

Comment:
consider adding tests with other numbers: 2, 8, 16

Reply:
Thank you for the review. I've updated PR based on your hints.